qutip/solve/mcsolve.py

```diff
@@ -276,7 +276,7 @@ def make_system(self, H, c_ops, tlist=None, args={}, options=None):
         ss.col_args = var
         for c in c_ops:
             cevo = QobjEvo(c, args, tlist=tlist)
-            cdc = cevo._cdc()
+            cdc = cevo.dag() * cevo
```

Comment:
Given the discussions happened around `__mul__` vs `__matmul__` should we use `cevo.dag() @ cevo` here?